source/components/Autocomplete.js

```diff
@@ -135,8 +135,6 @@ class AutocompleteBase extends Component<Props, State> {
       this.removeOption(this.state.selectedOptions.length - 1, event);
     } else if (event.keyCode === 13) { // Open suggestions on ENTER
       this.open();
-    } else if (event.keyCode === 27) { // Close suggestions on ESC
-      this.close();
```

Comment:
@daniloprates @DominikGuzei 
Hi guys, it seems both of you were working on `onKeyDown` in `Autocomplete` yesterday around the same time. I've tested both Danilo's branch that's open in [PR 70](https://github.com/input-output-hk/react-polymorph/pull/70/files) and Dominik's changes that have already been merged into develop. Danilo, you can see here that Dominik removes the check for the escape key and in your PR you add this same check to prevent the Modal from closing. Strangely enough, the Modal doesn't close when I press the escape key running Dominik's code even though it lacks this check: `event.keyCode === 27` . When I remove this check in Danilo's code, Modal does indeed close. I don't see any other changes in the commit history for either of you regarding this behavior so I'm pretty confused as to why it doesn't close on the develop branch but it does in PR 70 when the check for the escape key is removed.

Reply:
That's weird … i also tested it on this branch and i saw the issue with closing the modal - as expected! Maybe you have to clear the cache `npm run clear:cache` and/or re-install the node modules.